sql/src/main/java/io/crate/expression/symbol/SymbolVisitors.java

```diff
@@ -22,12 +22,22 @@
 
 package io.crate.expression.symbol;
 
+import java.util.List;
 import java.util.function.Predicate;
 
 public class SymbolVisitors {
 
     private static final AnyPredicateVisitor ANY_VISITOR = new AnyPredicateVisitor();
 
+    public static boolean any(Predicate<? super Symbol> symbolPredicate, List<Symbol> symbols) {
+        for (Symbol symbol : symbols) {
+            if (ANY_VISITOR.process(symbol, symbolPredicate)) {
```

Comment:
nitpicking (so feel free to ignore if you want to) - could call the other `any` method here (in case we change the logic of visiting _one_ symbol)

Reply:
actually this is what I wanted to do... but didn't for no reason ;) will change..